tools/cockpit.spec

```diff
@@ -69,6 +69,10 @@ Patch5:         storage-btrfs.patch
 Patch101:       hide-pcp.patch
 Patch102:       0002-selinux-temporary-remove-setroubleshoot-section.patch
 
+# For anything based on SLES 15 codebase (including Leap, SLE Micro)
+%if 0%{?suse_version} == 1500
```

Comment:
You don't need to wrap Patch: with %if. If the patches are unapplied, it's ok. But we can't have them missing because then we have errors with source validators at `osc commit` times.